Dockerfile.dev

```diff
@@ -27,7 +27,7 @@ COPY --from=horizon /go/bin/horizon /usr/bin/stellar-horizon
 
 COPY --from=friendbot /app/friendbot /usr/local/bin/friendbot
 
-COPY --from=soroban-rpc /app/soroban-rpc /usr/bin/stellar-soroban-rpc
```

Comment:
It copies it for the final image, but not for the build image. We're using the build image so that we don't have a dependency on apt-getting core. That's why this pr adds `--target build`, so we're only building the `build` target, and copying from that. That's what we do with Horizon as well.

Reply:
would it really be there or in `/go/bin/soroban-rpc` ?

Reply:
According to the docker file, it lives in /bin/soroban-rpc. See: https://github.com/stellar/soroban-tools/blob/main/cmd/soroban-rpc/docker/Dockerfile#LL5C17-L5C26